src/platform/plugins/shared/esql/public/triggers/esql_controls/esql_control_action.ts

```diff
@@ -14,8 +14,9 @@ import type { ISearchGeneric } from '@kbn/search-types';
 import type { ESQLVariableType, ESQLControlVariable } from '@kbn/esql-validation-autocomplete';
 import { monaco } from '@kbn/monaco';
 import type { ESQLControlState } from './types';
+import { isActionCompatible, executeAction } from './esql_control_helpers';
 
-const ACTION_CREATE_ESQL_CONTROL = 'ACTION_CREATE_ESQL_CONTROL';
+export const ACTION_CREATE_ESQL_CONTROL = 'ACTION_CREATE_ESQL_CONTROL';
```

Comment:
Export `ACTION_CREATE_ESQL_CONTROL` from a seperate file, like `constants.ts`. Otherwise `esql_control_action.ts` gets included in the page load bundle. Which is why the page load bundle is increasing this PR instead of decreasing.

Reply:
hmm I see, ok 🆗